dist/mlhr-table.js

```diff
@@ -568,11 +571,11 @@ angular.module('datatorrent.mlhrTable.directives.mlhrTable', [
       restrict: 'EA',
       replace: true,
       scope: {
-        columns: '=',
+        _columns: '=columns',
         rows: '=',
         classes: '@tableClass',
         selected: '=',
-        options: '=?',
+        _options: '=?options',
```

Comment:
See https://docs.angularjs.org/api/ng/service/$compile

If the parent scope property doesn't exist, it will throw a
NON_ASSIGNABLE_MODEL_EXPRESSION exception. You can avoid this behavior
using =? or =?attr in order to flag the property as optional.

On Monday, April 6, 2015, Andy Perlitch <notifications@github.com> wrote:

> In dist/mlhr-table.js
> <https://github.com/DataTorrent/malhar-angular-table/pull/40#discussion_r27813868>
> :
>
> >          rows: '=',
> >          classes: '@tableClass',
> >          selected: '=',
> > -        options: '=?',
> > +        _options: '=?options',
>
> did you check that this is the correct syntax? As opposed to _options:
> '=options?'? You're probably right, I just dont regularly see the
> optional syntax with an alternate attr being used...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/DataTorrent/malhar-angular-table/pull/40/files#r27813868>
> .
>